nemo_skills/pipeline/eval.py

```diff
@@ -176,6 +176,7 @@ def eval(
         "--not_exclusive",
         help="If --not_exclusive is used, will NOT use --exclusive flag for slurm",
     ),
+    with_sandbox: bool = typer.Option(False, help="If True, will start a sandbox container alongside this job"),
```

Comment:
Please update openmathreasoning eval docs to specify sandbox when running in TIR mode